ProjectGroupA/src/main/java/com/fundation/search/model/Search.java

```diff
@@ -72,9 +82,18 @@ public Search() {
 		BufferedReader in = new BufferedReader(new InputStreamReader(pDOS.getInputStream()));
 		String inputLine = "";
 		
+		if (!file.getSize().equals("0"))
```

Comment:
if statements always use braces even when it only has one-line. please set braces {}

Reply:
Done